docs/contributor/telemetry-internal-metrics.md

```diff
@@ -1,9 +1,18 @@
+<!-- The table below was generated automatically -->
+<!-- Please do not edit it directly. If you want to change it, please update the template and regenerate the file with make target  update-metrics-docs -->
 # Metrics Emitted by Kyma Telemetry Manager
 
-| Metric                                               | Description                                                                       |
-|------------------------------------------------------|:----------------------------------------------------------------------------------|
-| **telemetry_self_monitor_prober_in_flight_requests** | The current number of in-flight requests initiated by the self-monitoring prober. |
-| **telemetry_self_monitor_prober_requests_total**     | Total number of requests initiated by the self-monitoring prober.                 |
-| **telemetry_self_monitor_prober_duration_seconds**   | A histogram of latencies for requests initiated by the self-monitoring prober.    |
-| **telemetry_build_info**                             | Build information of the Telemetry Manager                                        |
-| **telemetry_feature_flags_info**                     | Enabled feature flags in the Telemetry Manager                                    |
+| Metric                                                          | Description                                                                                   |
+|-----------------------------------------------------------------|:----------------------------------------------------------------------------------------------|
+| **logs_generated_total** | Total number of logs generated |
+| **logs_generated_rate** | Actual rate of logs generated per second |
+| **loadgenerator_data_items_sent** | Number of data items sent by the load generator |
+| **mockbackend_data_items_received** | Number of data items received by the mock backend |
+| **telemetry_self_monitor_prober__duration_seconds** | A histogram of latencies for requests initiated by the self-monitoring prober. |
+| **telemetry_build_info** | Build information of the Telemetry Manager |
+| **telemetry_feature_flags_info** | Enabled feature flags in the Telemetry Manager |
+| **telemetry_pipelines_metric_pipeline_feature_usage** | Number of MetricPipelines using specific features |
+| **telemetry_pipelines_log_pipeline_feature_usage** | Number of LogPipelines using specific features |
+| **telemetry_pipelines_trace_pipeline_feature_usage** | Number of TracePipelines using specific features |
+| **telemetry_self_monitor_prober__in_flight_requests** | The current number of in-flight requests initiated by the self-monitoring prober. |
```

Comment:
Pls verify if the double underscore is correct. It was a single underscore before.

Reply:
You are right, it was a mistake. Done.